azure/rs_azure_compute/azure_compute_plugin.rb

```diff
@@ -181,6 +212,12 @@
       path "$href/instanceView"
     end
 
+    polling do
+     field_values do	 
+     end    
+       period 60
+	   action 'list_all'
+    end
     output "id","name","location","tags","properties","nextLink"
```

Comment:
We need to have a field called region for the UI. therefore we need to transform location to region in it's own output.  The same for all of the other resource types.  thx

Reply:
Also needs a status output.  Same for the other resources